### ISSUE TITLE
Training stays at maximum loss at or above 5000 global steps, nothing usable

### ISSUE THREAD

**Issue by XodrocSO** [rinongal/textual_inversion#20]
Running on Windows 10, RTX 3090, 128GB DDR4 and AMD Ryzen 9 5950X 16-core CPU.

Using "gloo" as the backend. Changed SIGUSR1(&2) to SIGTERM or I get an signal error.

I'm using the stable-diffusion v1-finetune.yaml in a repo that includes Stable diffusion, however I also tested on your repo and get the same result.

running with:

`python.exe main.py --base configs/stable-diffusion/v1-finetune.yaml -t --actual-resume models/ldm/stable-diffusion-v1/model.ckpt -n TestTrain --gpus 1, --data_root imagesets/Jn3`

init word is "person" defined in the yaml, string is still *

Base_learning_rate: 5.0e-03

The input images in the logs look fine but the reconstruction is just extreme noise, as is the others.. the conditioning image is just white with text(normal?)

At the start of the run I'm getting "RuntimeWarning: You are using LearningRateMonitor callback with models that have no learning rate schedulers. Please see documentation for configure_optimizers method.
rank_zero_warn(" from pytorch lightning lr_monitor.py

I've tried training with 56 images, 17, 5 and 3, didn't make a difference.
Here's a samples_scaled at 6500 global steps (same result at 60000 global steps as well)
![samples_scaled_gs-006500_e-000043_b-000050](https://user-images.githubusercontent.com/7324802/186782349-65f0f1ff-07a0-4670-94ad-1d4410607271.png)

**Comment by 1blackbar** [comment]
I dont think its normal, my reconstruction images are perfect right away without hardly any training, i think sample images are the ones that show actual prompted result, i have empty conditioning images as well, i did not adjust yaml file at all tho.

**Comment by XodrocSO** [comment]
@1blackbar are you running on Windows as well?  Definitely can't be normal.  My reconstruction images are mostly noise with some noticeable patterns if there's a reflective surface somewhere in the photo.

**Comment by 1blackbar** [comment]
yes im on windows, i train for  5 hours now and i can see some improvements on samples results, but i have like 22 k iterations and results are not as big ,  i think training should last a couple days if someone wants to have decent results.

**Comment by vasimr** [comment]
What is your step loss reported as? 
I was having a similar issue when I made a typo in the arguments (I used -actual_resume instead of --actual_resume). This gave me a consistent loss of around 1.0, while a correctly initialized run was around 0.13
It seems like there are no checks in place to verify that the pre-trained model is correctly loaded.

**Comment by XodrocSO** [comment]
@vasimr bingo!  Think you just solved my problem. a typo(which I think I just copy pasted from a second hand source) too much staying up late troubleshooting stuff to think.  I was getting loss 1.0 or 0.99, now starting at 0.13 as well!  Going to confirm it's fixed and then I'll close this!  Many thanks.

**Comment by XodrocSO** [comment]
> yes im on windows, i train for 5 hours now and i can see some improvements on samples results, but i have like 22 k iterations and results are not as big , i think training should last a couple days if someone wants to have decent results. Im on 1080ti, had to lower batch size from 2 to 1 so maybe im really on 10k results....

My problem is fixed now, but excited to see you got it working on a 1080ti..  Makes me wonder if I can get it working on my old GTX TITAN X.

Waiting to close this ticket until @rinongal see's it, as a check for typos or if the model is loaded will likely clear up up the issue for others encountering this too!

**Comment by rinongal** [comment]
I'll modify the actual_resume parameter to be required rather than optional. No real need for from-scratch training here and it should resolve this particular issue.
